src/components/ArticlePage/MarkdownHtml.js

```diff
@@ -9,35 +9,70 @@ import {
 } from '../../config';
 import { H4, H5, H6, P, Ul, Ol, Li, Hr, A, Strong, Em } from '../../components';
 
-require('prismjs/themes/prism-solarizedlight.css');
+require('prismjs/themes/prism-tomorrow.css');
 
 const HighlightStyle = createGlobalStyle`
+
+  // Decrease the distance between p and the code block
+  p + .gatsby-highlight {
+    margin-top: -${2 * baselineSmall}px;
+
+    @media (min-width: ${baselineBreakpoint}px) {
+      margin-top: -${2 * baselineLarge}px;
+    }
+  }
+
+  li p + .gatsby-highlight {
+
+    // Baseline offsets
+    margin-top: 0px;
+    margin-bottom: 18px;
+
+    @media (min-width: ${baselineBreakpoint}px) {
+
+      // Baseline offsets
+      margin-top: -2px;
+      margin-bottom: 18px;
+    }
+  }
+
   .gatsby-highlight  {
+    margin-bottom: ${2 * baselineSmall}px;
+
+    @media (min-width: ${baselineBreakpoint}px) {
+      margin-bottom: ${2 * baselineLarge}px;
+    }
 
     // Remove theme default styles
     pre[class*="language-"] {
-      background-color: transparent;
       margin: ${baselineSmall}px 0;
-      padding: ${2 * baselineSmall}px 0;
+      padding: ${2 * baselineSmall}px 16px;
       line-height: 24px;
-      border-radius: 0;
+      border-radius: 4px;
 
       @media (min-width: ${baselineBreakpoint}px) {
         margin: ${baselineLarge}px 0;
-        padding: ${2 * baselineLarge}px 0;
+        padding: ${2 * baselineLarge}px 24px;
       }
     }
     code[class*="language-"] {
       line-height: 24px;
+      font-size: 14px;
+      background: none;
+      font-weight: 400;
+
+      @media (min-width: ${baselineBreakpoint}px) {
+        font-size: 16px;
+      }
     }
 
     pre[class*="language-"],
     code[class*="language-"],
     .token {
       &::selection {
         // Use same selection styles as in sanitize.css
-        background-color: #b3d4fc;
```

Comment:
You could remove the comment above as the styles don't come from the sanitize.css defaults anymore.